test-jest/chart/AreaChart.spec.tsx

```diff
@@ -55,50 +55,37 @@ describe('AreaChart', () => {
     });
   });
 
-  test.skip('Renders customized active dot when activeDot is set to be a ReactElement', () => {
+  test.skip('Renders customized active dot when activeDot is set to be a ReactElement', async () => {
```

Comment:
Would you like to try to also get the test to work? Or would you prefer to do that in a separate PR?

Reply:
I think this skip was because of the `jsdom` limitations (can't find things on mouse enter/hover